config.yml

```diff
@@ -772,6 +772,16 @@ nodes:
 
           module Foo end
           ^^^^^^^^^^^^^^
+  - name: MultiLeftHandNode
+    child_nodes:
+      - name: targets
```

Comment:
We called these "targets" because in the assign nodes, the variables being assigned are also called "target". Happy to change the name to whatever makes more sense.

Reply:
Targets is good!